README.md

```diff
@@ -1,3 +1,104 @@
-# secure-api-gateway-releases
-This repo holds the versioned helm and kustomize objects to be able to deploy the third party sources and secure-api-gateway
+# Secure API Gateway - Releases
 
+## Prerequisites
+
+- Kubernetes v1.23 +
+- Helm 3.0.0 +
+
+To add the forgerock helm artifactory repository to your local machine to consume helm charts use the following;
+
+```console
+  helm repo add forgerock-helm https://maven.forgerock.org/artifactory/forgerock-helm-virtual/ --username [backstage_username]  --password [backstage_password]
+  helm repo update
+```
+
+NOTE: You must have a valid [subscription](https://backstage.forgerock.com/knowledge/kb/article/a57648047#XAYQfS) to aquire the `backstage_username` and `backstage_password` values.
+
+It is assumed that, the following components have been sucessfully built and docker images are within a docker container registry ready to be used. 
+
+- [Secure API Gateway - Remote Consent Service](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-rcs)
+- [Secure API Gateway - Remote Consent Service User Interface](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-ui)
+- [Secure API Gateway - Test Bank Facility](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-rs)
+- [Secure API Gateway - Test User Account Creator](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-test-data-initializer)
+
+It is assumed that, either a deployments repo has been created containing the specific deployment overrides has been created, or if a repository are not being used they are at hand (within the instructions below it is assumed that a repo is being used)
+
+It is assumed that all infrastructure has been created, including deploying the Cloud Instance, and that the following prerequisits have been deployed to the cluster
+- [Nginx](https://kubernetes.github.io/ingress-nginx/deploy/)
+- [MongoDB](https://github.com/SecureApiGateway/secure-api-gateway-releases/blob/master/third-party/Chart.yaml)
+- [Secrets](https://github.com/SecureApiGateway/secure-api-gateway-releases-helpers/tree/master/external-secrets-gsm) 
+  NOTE: GSM has been used but is not manditory, other secrets managers can be used.
+
+## Deployment Guide [Internal]
+
```

Comment:
This should be documented in the internal wiki so as not to confuse 'customers' of the project. Really, you're documenting how to use the Action, so by all means put a README in github workflows.

Reply:
Done! Moved to Readme in workflows